packages/components/addon/components/hds/dropdown/toggle-user.hbs

```diff
@@ -1,11 +1,11 @@
 <button
   class={{this.classNames}}
-  ...attributes
   aria-label={{@text}}
   type="button"
   disabled={{if this.isDisabled "disabled" null}}
+  ...attributes
```

Comment:
[chore] you don't need attributes spread here, if you look at how it's used in the `index` there are not attributes applied: https://github.com/hashicorp/design-system/blob/44d75fa7d74b9a6a85a6924a79a224f3cd93303f/packages/components/addon/components/hds/dropdown/index.hbs#L4-L9

Reply:
we'll make a decision on this later, we may want to use a different pattern (pass the components as yielded elements in the block)